blueprint/index.md

```diff
@@ -7,232 +7,285 @@ icon: blueprint
+* **Amazon API Gateway** - An AWS service for using APIs in a secure and scalable environment. In this solution, the API Gateway exposes a REST endpoint that is protected by an API key. Requests that come to the API Gateway are forwarded to an AWS Lambda.
+* **REST API client** - A method or tool used to invoke REST API services that are exposed for communication. In this solution, you send an HTTP PUT request with JSON payload to the Bot Connector API that creates the bot list. For example, Postman, YARC or any other tool to send REST API requests and receive responses.
+* **Website with Messenger** - A website with a message window to interact with the bot. It is required to test the solution.
+  
 
 ## Prerequisites
 
 ### Specialized knowledge
 
-[comment]: # List specialized knowledge. Copy appropriate items from library, if description exists. Otherwise, follow format and brevity of descriptions in library. Examples follow. &&: we need to put library in public space.
+* Administrator-level knowledge of Genesys Cloud
+* Experience using the Genesys Cloud API
+* Experience using Microsoft Power VA
+* Experience using Architect flows
+* Knowledge about adding scripts to HTML pages
+
+### Genesys Cloud account
 
-* Administrator-level knowledge of Genesys Cloud.
-* Experience with using the Genesys Cloud API.
-* Experience with Power VA from Microsoft.
-* Ability to create Architect flows.
-* Ability to add scripts to HTML pages.
+* A Genesys Cloud license. For more information, see [Genesys Cloud pricing](https://www.genesys.com/pricing "Opens the Genesys Cloud pricing page") on the Genesys website.
+* The Master Admin role in Genesys Cloud. For more information, see [Roles and permissions overview](https://help.mypurecloud.com/?p=24360 "Opens the Roles and permissions overview article") in the Genesys Cloud Resource Center.
 
-### {Type of} account
+### AWS account
 
-[comment]: # Add separate H3 sections for each account, for example, Genesys Cloud account, Vonage account, Salesforce account. Example text follows.
+* An administrator account with permissions to access the following services:
+	* AWS API Gateway
+	* AWS Lambda
 
-* A Genesys Cloud license. For more information, see [Genesys Cloud pricing](https://www.genesys.com/pricing "Opens the Genesys Cloud pricing page") on the Genesys website.
-* (Recommended) The Master Admin role in Genesys Cloud. For more information, see [Roles and permissions overview](https://help.mypurecloud.com/?p=24360 "Opens the Roles and permissions overview article") in the Genesys Cloud Resource Center.
-* AWS account with billing to create Lambda functions and API Gateways
-* Azure account with billing to be able to create the Power VA agent
+### Microsoft Azure account
+* Admininstrator-level role for Microsoft Azure to create the bot in Power VA.
```

Comment:
To be consistent, could we say, "Aan administrator account with  permissions to create a bot in Microsoft Power VA"?